java-monitoring/run_protoc.sh

```diff
@@ -0,0 +1,814 @@
+#!/usr/bin/env bash
+
+LIBRARY_GEN_OUT=$(pwd)/../library_gen_out
+
+mkdir -p "${LIBRARY_GEN_OUT}"/gapic_generator_java/rules_java_gapic
+mkdir -p "${LIBRARY_GEN_OUT}"/com_google_protobuf/descriptor_proto
+mkdir -p "${LIBRARY_GEN_OUT}"/com_google_protobuf/duration_proto
+mkdir -p "${LIBRARY_GEN_OUT}"/com_google_protobuf/any_proto
+mkdir -p "${LIBRARY_GEN_OUT}"/com_google_protobuf/timestamp_proto
+mkdir -p "${LIBRARY_GEN_OUT}"/com_google_protobuf/struct_proto
+mkdir -p "${LIBRARY_GEN_OUT}"/com_google_protobuf/empty_proto
+mkdir -p "${LIBRARY_GEN_OUT}"/com_google_protobuf/field_mask_proto
+mkdir -p "${LIBRARY_GEN_OUT}"/com_google_protobuf/wrappers_proto
+mkdir -p "${LIBRARY_GEN_OUT}"/google/api
+mkdir -p "${LIBRARY_GEN_OUT}"/google/rpc
+mkdir -p "${LIBRARY_GEN_OUT}"/google/type
+mkdir -p "${LIBRARY_GEN_OUT}"/google/monitoring/v3
+mkdir -p "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-resources
+mkdir -p "${LIBRARY_GEN_OUT}"/google/monitoring/v3/google-cloud-monitoring-v3-java-resources
+mkdir -p "${LIBRARY_GEN_OUT}"/google/monitoring/v3/grpc-google-cloud-monitoring-v3-java-resources
+mkdir -p "${LIBRARY_GEN_OUT}"/google/monitoring/v3/proto-google-cloud-monitoring-v3-java-resources
+
+##################### Section 0 #####################
+# prepare tooling
+#####################################################
+# proto files from googleapis repository
+REPO_ROOT=$(pwd)/..
+cd "${REPO_ROOT}" || exit
```

Comment:
These many `|| exit` is redundant. Would you make this shell script to fail upon any command failure.

Reply:
Sure, will do.